alias_service.py

```diff
@@ -0,0 +1,25 @@
+from model import Player
+
+def get_top_suggestion_for_aliases(dao, aliases):
+    return dict( [ (entry[0], min(entry[1], key=lambda p: len(p.name))) for entry in get_player_suggestions_from_player_aliases(dao, aliases).iteritems() ] )
```

Comment:
This is a really complicated expression. I would prefer not to make it a one-liner.